opensearch-operator/pkg/builders/cluster_test.go

```diff
@@ -54,6 +54,18 @@ func ClusterDescWithAdditionalConfigs(addtitionalConfig map[string]string, boots
 var _ = Describe("Builders", func() {
 
 	When("Constructing a STS for a NodePool", func() {
+		It("should include the init containers", func() {
+			var clusterObject = ClusterDescWithVersion("2.2.1")
+			var result = NewSTSForNodePool("foobar", &clusterObject, opsterv1.NodePool{}, "foobar", nil, nil, nil)
+			Expect(len(result.Spec.Template.Spec.InitContainers)).To(Equal(1))
+		})
+		It("should skip the init containers", func() {
```

Comment:
Please extend the description a bit to make clear what is tested. Ala "should skip the init container as SKIP_INIT_CONTAINER is set".